packages/ckeditor5-dev-ci/lib/trigger-circle-build.js

```diff
@@ -5,7 +5,7 @@
 
 /* eslint-env node */
 
-const fetch = require( 'node-fetch' );
+import nodeFetch from 'node-fetch';
```

Comment:
I think it's time to drop `node-fetch` in favor of the native library while we upgrade dependencies.